reference/conanfile/tools/gnu/autotoolstoolchain.rst

```diff
@@ -232,3 +232,46 @@ The ``environment()`` method returns an :ref:`Environment<conan_tools_env_enviro
             env = at.environment()
             env.define("FOO", "BAR")
             at.generate(env)
+
+
+Customizing the xxxxxx_args attributes
+++++++++++++++++++++++++++++++++++++++
+
+``AutotoolsToolchain`` provides some methods to help users to add/update/remove values defined in ``configure_args``,
+``make_args`` and ``autoreconf_args`` (all of them lists of strings). Those methods are:
+
+* ``update_configure_args(updated_flags)`` (new since Conan 1.57): will change ``AutotoolsToolchain.configure_args``.
+* ``update_make_args(updated_flags)`` (new since Conan 1.57): will change ``AutotoolsToolchain.make_args``.
+* ``update_autoreconf_args(updated_flags)`` (new since Conan 1.57): will change ``AutotoolsToolchain.autoreconf_args``.
```

Comment:
```suggestion
* ``update_configure_args(updated_flags)`` (since Conan 1.57): will change ``AutotoolsToolchain.configure_args``.
* ``update_make_args(updated_flags)`` (since Conan 1.57): will change ``AutotoolsToolchain.make_args``.
* ``update_autoreconf_args(updated_flags)`` (since Conan 1.57): will change ``AutotoolsToolchain.autoreconf_args``.
```